packages/hoppscotch-common/src/components/collections/ImportExport.vue

```diff
@@ -435,26 +435,15 @@ const HoppTeamCollectionsExporter: ImporterOrExporter = {
       )
 
       if (E.isRight(res)) {
-        const { exportCollectionsToJSON } = res.right
```

Comment:
While bulk exporting collections from a team workspace, previously it didn't conform with the `HoppCollection` format and a re-import resulting in the collection level properties (authorization/headers) missing (one of the issues mentioned in the PR description as being patched). Hence, Instead of an object, the `teamCollectionsExporter` helper function (consumes `getTeamCollectionJSON()` behind the scenes) now returns the stringified contents straightaway. The format is changed since during bulk exports the JSON string is parsed and each entry undergoes transformation to the `HoppCollection` format and a stringified representation is returned.